packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextLayoutManagerMapBuffer.java

```diff
@@ -590,6 +668,8 @@ public static WritableArray measureLines(
         TextAttributeProps.getTextBreakStrategy(
             paragraphAttributes.getString(PA_KEY_HYPHENATION_FREQUENCY));
 
+    // TODO: adjustFontSizeToFit
```

Comment:
This is missing?

Reply:
Yes, I didn't want to repeat everything twice while testing, added in https://github.com/facebook/react-native/pull/44075/commits/e2bba84a5c63564fbb439722cbac8f49995b05e0.